packages/console/src/components/Sidebar/icons/Box.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Should we still prefer to explicitly import React since React now has a new jsx transform feature?
Check this out: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Reply:
yes but VSCode will complain. let me try

Reply:
Yes, I saw that too, will have to override the lint rules

Reply:
added LOG-1662